sdk/resources/Microsoft.Azure.Management.Resource/src/generate.ps1

```diff
@@ -5,8 +5,8 @@
 # Generate package with resources tag
 # Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "v2" -AutoRestCodeGenerationFlags "--tag=package-resources-2021-04" -SdkGenerationDirectory "$PSScriptRoot\Generated\Resources"
 
-# Generate package with templatespecs tag
-Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "v2" -AutoRestCodeGenerationFlags "--tag=package-templatespecs-2021-05" -SdkGenerationDirectory "$PSScriptRoot\Generated\TemplateSpecs"
+# Generate package with features tag
+Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "v2" -AutoRestCodeGenerationFlags "--tag=package-features-2021-07" -SdkGenerationDirectory "$PSScriptRoot\Generated\Features"
```

Comment:
Looks like we are replacing the SDK of templatespec with features? This would be a very big breaking change, could you please confirm this is exactly what you want?